server/src/models/index.ts

```diff
@@ -3,4 +3,4 @@ export * from './planter.model';
 export * from './species.model';
 export * from './adminUser.model';
 export * from './tag.model';
-export * from './treeTag.model';
+export * from './tree-tag.model';
```

Comment:
Looks like a little different from the convention of naming. Why we must name the model name with dash?
And I noticed that you change underscore to dash, we have convention for that?

Reply:
This is the style used by the Loopback v4 CLI for multi-word models/endpoints.
I thought it easiest to go along with it, but I'm happy to revert back to the old format for consistency.
What do you think?

Reply:
I'll change back to match the style of other filenames and endpoints.
There's nothing worse than inconsistency.